HesoyamHospital/IntegrationAdapter/SFTPServiceSupport/SFTPService.cs

```diff
@@ -1,26 +1,66 @@
 ﻿using Renci.SshNet;
 using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace IntegrationAdapter.SFTPServiceSupport
 {
     public class SFTPService
     {
+        private static readonly string serverIP = "192.168.1.92";
```

Comment:
Server IP should not be hardcoded like this. 
When we decide to deploy this application, we will have to change it.
Use enviorment variables instead, if possible.